h/static/styles/partials-v2/_nav-bar.scss

```diff
@@ -87,7 +92,7 @@
 @media screen and (max-width: $max-phone-width) {
   // Display search bar full-width underneath nav links on phone-sized screens
   .nav-bar {
-    height: 90px;
+    height: 120px;
```

Comment:
I did because in the mockup it looked like there should be 15px ish of margin betwen the search input box and the lozenges.